Source/DafnyCore/AST/Statements/AssignSuchThatStmt.cs

```diff
@@ -8,6 +9,10 @@ public class AssignSuchThatStmt : ConcreteUpdateStatement, ICloneable<AssignSuch
   public readonly Expression Expr;
   public readonly AttributedToken AssumeToken;
 
+  public override IEnumerable<Node> PreResolveChildren =>
+  Lhss.Concat<Node>(new List<Node>() {
+  Expr });
```

Comment:
The formatting of this declaration is a bit weird. Maybe indent line 13 and join line 14 with it?